add brightcove plugin docs

jeanstemp opened the pull request:
Adding a new plugins sections under dev-docs for the new Brightcove plugin.

jeanstemp commented:
@mkendall07 , @bretg , could one of you take a look at the PR and make sure everything looks okay in terms of placement and formatting? Thanks!

bretg commented on dev-docs/plugins/bc/bc-prebid-plugin-prebid-options.md:
```diff
@@ -0,0 +1,302 @@
+- [enablePrebidCache](#enablePrebidCache)
+
+<a name="prebidPath"></a>
+### prebidPath
```
Needs to be a space between the <a> and the ### here

jeanstemp replied:
@bretg I'm looking at it on my local build and I'm not seeing where the issue is, everything looks fine on my end. What are you seeing when you do your local build?

bretg commented on dev-docs/plugins/bc/bc-prebid-plugin-prebid-options.md:
```diff
@@ -0,0 +1,302 @@
+`options.prebidPath = 'http://your-path/prebid.js'`
+
+<a name="biddersSpec"></a>
+### biddersSpec
```
Needs to be a space between <a> and ### here

bretg commented on dev-docs/plugins/bc/bc-prebid-plugin-prebid-options.md:
```diff
@@ -0,0 +1,302 @@
+See [Setting Up Prebid Parameters](#set-up-params) below
+
+<a name="bidderSettings"></a>
+### bidderSettings
```
and here

bretg commented on dev-docs/plugins/index.md:
```diff
@@ -0,0 +1,31 @@
+
+# Prebid Plugins
+
+Prebid plugins allow you to extend your header bidding solutions.
```
Please describe the general concept of plugins more. I was initially confused about the difference between a plugin and a module. After reading it, I wonder if there will ever be any more plugins -- maybe this is a Brightcove-specific thing?

Maybe something like "A 'plugin' is Prebid presence in another technology platform."

vpitts commented:
@bretg The definition of a plugin is:  "a software component that adds a specific feature to an existing computer program".  Its purpose is to customize the behavior of the product where the plugin is being added.  In the case of the Brightcove Prebid Plugin, the plugin is a software component, consisting of several modules, that it designed to be added to the Brightcove Player to support invoking prebid.js and then to render the resulting winning ad in the Brightcove Player.

In the introductory document (about-bc-prebid-plugin) of the set of documents describing the Brightcove Prebid Plugin, we already have the statement:  "The Brightcove Plugin for Prebid (BcPrebidVast) is a plugin to the Brightcove player. This plugin will provide prebid support and then manage the ad playback of the ad that was selected via prebid, as supported by prebid.js via prebid.org.".  Is this sufficient or do you want us to add a blurb about the general definition of a plugin?

@jeanstemp What do you think?  Should we simply add text there about the definition of a plugin?

jeanstemp commented:
Thanks @vpitts for the great explanation of plugins! I've updated the intro page to hopefully provide a better description for customers. @bretg , @mkendall07 , @mjacobsonny Let me know if you think this works better.

jeanstemp commented:
I updated the description per @vpitts feedback, and changed the location and section naming based on @bretg feedback. Thanks all for the feedback, please review (hopefully) one last time!

vpitts commented:
I do approve the new changes.  So from my vantage, the doc is good to go

bretg commented:
confirmed that the ### formatting isn't visible in production. Lovely. Will upgrade my jekyll install.
